src/AcceptanceTests.SqlServer/MultiSchema/When_custom_schema_configured_for_endpoint.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace AcceptanceTests.SqlServer.MultiSchema
+{
+    using System.Threading.Tasks;
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.Features;
+    using NServiceBus.Transport.SqlServer;
+    using NUnit.Framework;
+    using Conventions = NServiceBus.AcceptanceTesting.Customization.Conventions;
+
+    public class When_custom_schema_configured_for_endpoint : BridgeAcceptanceTest
+    {
+        public const string PublisherSchema = "publisher";
+
+        [Test]
+        public async Task Subscriber_should_get_the_event()
+        {
+            var context = await Scenario.Define<Context>()
+            .WithBridge(bridgeConfiguration =>
+            {
+                var endpointAddress = GetTestEndpointAddress<Publisher>();
```

Comment:
I used this to test and see if the endpointAddress would have the `Publisher` class `defaultSchema` value set, but it does not.